core/rlp_test.go

```diff
@@ -195,3 +200,35 @@ func BenchmarkHashing(b *testing.B) {
 		b.Fatalf("hash wrong, got %x exp %x", got, exp)
 	}
 }
+
+func TestBlockRlpEncodeDecode(t *testing.T) {
```

Comment:
Is there already a roundtrip RLP encoding test for blocks someplace in this codebase? What does this cover specifically that would not be covered otherwise?

Reply:
There is an existing RLP encoding roundtrip that is part of a test that only checks whether transactions/uncles can be iterated -- and it uses `TestChainConfig` that does not have op-stack forks set. This test aims to exercise the behavior of creating a new block when Isthmus hard fork is active: https://github.com/ethereum-optimism/op-geth/pull/399/files#diff-e177ddb5610d1f3426c15cb512999613785b20838954e48187883e6263bb282bR299-R304 and making sure that the new header field is as expected after the encode/decode round trip.